packages/firebase_auth/firebase_auth_platform_interface/test/platform_interface_tests/platform_interface_auth_test.dart

```diff
@@ -333,6 +333,16 @@ void main() {
       );
     });
 
+    test('throws if .useEmulator', () {
+      try {
+        firebaseAuthPlatform.useEmulator('http://localhost', 9099);
+      } on UnimplementedError catch (e) {
+        expect(e.message, equals('useEmulator() is not implemented'));
+        return;
+      }
+      fail('Should have thrown an [UnimplementedError]');
```

Comment:
This should use `expectLater` like the other tests in this file

Reply:
Nice catch! That's exactly the sort of Dart-error I would expect to make and I'm glad you've got more experienced 👀  to see that stuff, I'll fix and re-push

Reply:
fixed, pushed